code/harness/c_harness.cpp

```diff
@@ -139,7 +138,38 @@ int main(int argc, char** argv) {
    ****************************************************************************/
   const unsigned int test_case_idx = optind;
   const string path = argv[test_case_idx];
-   
```

Comment:
leave newline here instead of removing it and please remove commented out code that was used for development